colossus/src/main/scala/colossus/controller/OutputController.scala

```diff
@@ -285,4 +291,13 @@ trait OutputController[Input, Output] extends MasterController[Input, Output] {
     }
   }
 
+  def idleCheck(period: Duration): Unit = {
```

Comment:
Yeah these methods are only invoked by the worker and are thus threadsafe.  In fact, there is no code in this class or any other class inheriting `ConnectionHandler` that is invoked anywhere outside the worker.